main.go

```diff
@@ -36,6 +36,7 @@ func whoAmI() string {
 
 func main() {
 	log.SetFlags(log.LstdFlags | log.Lshortfile)
+	location := flag.String("l", "localhost", "IPv4 address to listen on")
```

Comment:
This isn't just going to accept an "ipv4 address" - it'll accept ipv6 addresses and hostnames too? I'm thinking we should say something like "address to listen on" instead

Reply:
Actually this isn't that simple, IPv6 addresses need to be enclosed in square brackets before being passed to [net.Dial](https://pkg.go.dev/net#Dial)...